phoenicis-javafx/src/main/java/org/phoenicis/javafx/utils/SwitchBindingBuilder.java

```diff
@@ -0,0 +1,162 @@
+package org.phoenicis.javafx.utils;
+
+import com.google.common.collect.ImmutableMap;
+import javafx.beans.Observable;
+import javafx.beans.binding.ObjectBinding;
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.SimpleObjectProperty;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * A builder class for <code>switch</code> like {@link ObjectBinding}s
```

Comment:
I think this should be put outside phoencis (can be done later). 
Also, you need to provide simple examples to see how it works exactly.
Overall, it looks clean

Reply:
You're right about that. I'm planning to move the whole utils package to a new repository later on. (Similar to the collections)